diff --git a/dist/index.js b/dist/index.js
index e21b4d2644991efdbeacfd00935fb38040ed45d6..019c552f5963033167f683df80a339c08e3176e5 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -2631,7 +2631,8 @@ class TabsAPI {
       this.onRemoved(tabId);
     });
     this.onCreated(tabId);
-    this.onActivated(tabId);
+    const activeTab = this.ctx.store.getActiveTabFromWebContents(tab)
+    if (activeTab?.id === tabId) this.onActivated(tabId);
     debug(`Observing tab[${tabId}][${tab.getType()}] ${tab.getURL()}`);
   }
 
